config/variables.go

```diff
@@ -0,0 +1,161 @@
+package config
+
+import (
+	"fmt"
+	"github.com/gruntwork-io/boilerplate/util"
+	"github.com/gruntwork-io/boilerplate/errors"
+	"strings"
+	"io/ioutil"
+	"gopkg.in/yaml.v2"
+)
+
+// Get a value for each of the variables specified in boilerplateConfig. The value can come from the user (if the
+// non-interactive option isn't set), the default value in the default value in the config, or a command line option.
```

Comment:
s/the default value in the default value in the config/the default value in the config/